conduit/scripts/localwrap

```diff
@@ -37,39 +38,52 @@ parser = argparse.ArgumentParser(
 
 parser.add_argument('--input', help='Comma delimited list of files to download in the form /directory/file')
 parser.add_argument('--output', help='Comma delimited list of files to upload in the form /directory/file')
+parser.add_argument('--wf', help = 'Workflow name in the form of saber_bucket/wf_hash')
 parser.add_argument('command', nargs=argparse.REMAINDER, help='The actual command being run form of command args ...')
 
 args = parser.parse_args()
+if args.wf:
+    volume_path = '../volumes/data/local/{}/'.format(args.wf)
```

Comment:
we should remember to open an issue to check the local filesystem, or put some kind of flag in for windows file systems. The trick is how to do this from within the docker container.